src/scripts/components/globe/globe.tsx

```diff
@@ -295,6 +299,8 @@ const Globe: FunctionComponent<Props> = ({
     flyToGlobeView(viewer, flyTo);
   }, [viewer, flyTo]);
 
+  useMarkers(viewer, stories);
```

Comment:
If the logic is that we only show the markers when there is no story or data layer, I think a clearer solution would be to give the globe a property 'markers' and pass some marker data when there is no story pr data layer active. Sorry that my first guess was to use a flag in the redux store but with this behaviour it doesn't seem like a robust solution anymore.

We can talk about this in person if you want to...